### PR TITLE
Added some documentation to classes to make it easier to understand.

### PR DESCRIPTION
**Pull request by jackflip**
I added some documentation to some recently written methods.

### CODE REVIEW

File: src/main/java/me/ironexception/universalfriends/association/Association.java [hunk 1]
```diff
@@ -38,8 +41,14 @@ public double getValue() {
     /**
      * Fetch a {@link Association} by its value.
      * Might return null if there is no corresponding
-     * @param value
-     * @return
+     *
+     * <p>
+     * This is like the friendliness level.
```
Comment (zeroeightysix) [inline src/main/java/me/ironexception/universalfriends/association/Association.java]: It's not just *like* the friendliness level, it really **is** the friendliness *value*.

File: src/main/java/me/ironexception/universalfriends/association/Association.java [hunk 2]
```diff
@@ -38,8 +41,14 @@ public double getValue() {
...
+     * </p>
+     *
+     * @param value THe friendliness value closest to the actual relation.
```
Comment (zeroeightysix) [inline src/main/java/me/ironexception/universalfriends/association/Association.java]: T*H*e

**Review by zeroeightysix — CHANGES_REQUESTED** [review]

File: src/main/java/me/ironexception/universalfriends/association/Association.java (hunk 2 above)
Comment (jackflip) [reply src/main/java/me/ironexception/universalfriends/association/Association.java]: fixing it as we speak

File: src/main/java/me/ironexception/universalfriends/association/Association.java [hunk 3]
```diff
@@ -38,8 +41,14 @@ public double getValue() {
...
      * Fetch a {@link Association} by its value.
      * Might return null if there is no corresponding
-     * @param value
-     * @return
+     *
+     * <p>
+     * This is like the friendliness level.
+     * </p>
+     *
+     * @param value The friendliness value closest to the actual relation.
```
Comment (zeroeightysix) [inline src/main/java/me/ironexception/universalfriends/association/Association.java]: > closest to the actual relation.

Hmm, this function will return `null` for anything that isn't *exactly equal* to an association's assigned value. But how do we word that?

Comment (jackflip) [reply src/main/java/me/ironexception/universalfriends/association/Association.java]: I say, we cap off the value in the method and actually prevent that nullification from actually happening. I'll write it up real quick and see what you think.

Comment (zeroeightysix) [reply src/main/java/me/ironexception/universalfriends/association/Association.java]: > I say, we cap off the value in the method and actually prevent that nullification from actually happening. I'll write it up real quick and see what you think.

Do that in a separate method. I would, as a developer, only expect this method to return an `Association` if the value matches exactly the association's value. We don't want any magic to happen behind the scenes here.

**Review by zeroeightysix — APPROVED** [review]

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**